src/hooks/@common/usePrivateRecommendations.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * 친구 찾아보기 - 로그인 상태
+ */
+
+import { getPrivateUserRecommendations } from '@/apis/@common';
+import { COMMON_KEYS } from '@/constants/@queryKeys';
+import isLoggedInAtom from '@/recoil/auth/isLoggedIn';
+import { useQuery } from '@tanstack/react-query';
+import { useRecoilValue } from 'recoil';
+
+const usePrivateRecommendations = (size: number) => {
+  const isLoggedIn = useRecoilValue(isLoggedInAtom);
+
+  return useQuery({
+    queryKey: COMMON_KEYS.RECOMMENDED_USERS,
+    queryFn: () => getPrivateUserRecommendations(size),
+    enabled: size !== 0 || !!isLoggedIn,
```

Comment:
```suggestion
    enabled: size !== 0 && !!isLoggedIn,
```
로그인 상태이고, size가 0이 아닐 때 데이터를 불러와야 할 것 같은데 `||` 로 구현하신 이유가 있으실까요?

Reply:
앗 해당 부분 논리 연산자는 OR이 아니라 AND가 맞겠네요! 👍👍